### ISSUE TITLE
Y7S3 files are not supported

### ISSUE THREAD

**Issue by DudeKiller82** [parzivail/RainbowForge#25]
The tool need an update to support the new format.

I really need your tool.

Good luck !

**Comment by luqmanghost18** [comment]
Yes, the tools are so good but Ubi have patched it and need a fix/update since Y7S2. Hope anyone can give help on updating this. Thanks.
